CHANGELOG.rst

```diff
@@ -10,8 +10,19 @@
  | # # # for more details. If you did not receive the license, for more information see:
  | # # # https://github.com/U-S-NRL-Marine-Meteorology-Division/
 
-Please see geoips/CHANGELOG_TEMPLATE.rst for instructions on updating
-CHANGELOG appropriately with each PR
+Bug Fixes
```

Comment:
You can put this directly in v1_11_1a0.rst if you want.

Reply:
Done and I reverted CHANGELOG.rst.